ml-algorithms/src/test/java/org/opensearch/ml/engine/memory/MLMemoryManagerTest.java

```diff
@@ -0,0 +1,124 @@
+package org.opensearch.ml.engine.memory;
+
+import static org.junit.Assert.*;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.ArgumentMatchers.anyString;
+import static org.mockito.Mockito.doNothing;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+
+import java.util.List;
+import java.util.Map;
+
+import org.junit.Before;
+import org.junit.Test;
+import org.mockito.Mock;
+import org.mockito.MockitoAnnotations;
+import org.opensearch.action.update.UpdateResponse;
+import org.opensearch.client.AdminClient;
+import org.opensearch.client.Client;
+import org.opensearch.client.IndicesAdminClient;
+import org.opensearch.cluster.ClusterState;
+import org.opensearch.cluster.metadata.Metadata;
+import org.opensearch.cluster.service.ClusterService;
+import org.opensearch.common.settings.Settings;
+import org.opensearch.common.util.concurrent.ThreadContext;
+import org.opensearch.core.action.ActionListener;
+import org.opensearch.ml.common.conversation.Interaction;
+import org.opensearch.ml.memory.action.conversation.CreateConversationResponse;
+import org.opensearch.ml.memory.action.conversation.CreateInteractionResponse;
+import org.opensearch.ml.memory.index.ConversationMetaIndex;
+import org.opensearch.threadpool.ThreadPool;
+
+public class MLMemoryManagerTest {
```

Comment:
Should we delete this class as Xun's PR include this test class and merged to main

Reply:
Yes it can be deleted or added these tests into my test class which is MLMemoryManagerTests. Only difference in the test class name is the last letter "s".

Reply:
@Zhangxunmt can you help merge or delete this in your next PR ?